test/it/client-listUsers.js

```diff
@@ -25,7 +41,7 @@ describe('client.listUsers()', () => {
   });
```

Comment:
How will the test-server handle matching requests (like `DELETE /api/v1/users/{id}` where an arbitrary ID is included in the request?